public/js/components/Editor.js

```diff
@@ -49,6 +49,9 @@ const Editor = React.createClass({
       gutters: ["breakpoints"]
     });
 
+    // NOTE: only used for testing
+    window.cm = this.editor;
```

Comment:
sounds good. I remember doing that as well, although that might be how we're doing it in Firefox. I agree about the littering aspect. I was thinking about namespacing them or wrapping them in a function like `addDebugGlobal` but figured I'd wait